pulse_actions/publisher.py

```diff
@@ -16,10 +16,10 @@
 
 
 class ExperimentalPublisher(GenericPublisher):
-    def __init__(self, **kwargs):
+    def __init__(self, user, **kwargs):
         super(ExperimentalPublisher, self).__init__(
             PulseConfiguration(**kwargs),
-            'exchange/adusca/experiment',
+            'exchange/%s/pulse_actions' % user,
```

Comment:
If this is really a problem, you can kwargs.update({'user': user}) before passing it as an argument to PulseConfiguration.